completion/available/pip.completion.bash

```diff
@@ -7,5 +7,13 @@
 # you should first initialize the corresponding environment.
 # So that pip is in the system's path.
 if _command_exists pip; then
-	eval "$(pip completion --bash)"
+	function _pip_lazy() {
```

Comment:
Most popular tools follow the pattern `__<tool with underscores>_complete_<function name>` and I think we should to.

```suggestion
	function __bash_it_complete_pip() {
```